src/lib.rs

```diff
@@ -128,6 +128,10 @@
                   cast_possible_truncation, cast_precision_loss,
                   shadow_reuse, cyclomatic_complexity, similar_names,
                   doc_markdown, many_single_char_names))]
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Typically this is an unconditional `#![no_std]` if a crate is no_std compatible, but I'll send a PR for this.